extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties

```diff
@@ -0,0 +1,130 @@
+thing-type.sonos.PLAYBAR.label = PLAYBAR
+thing-type.sonos.PLAYBAR.description = Dient zur Steuerung von Sonos PLAYBAR Speakern.
+thing-type.sonos.zoneplayer.label = Gruppen Player
+thing-type.sonos.zoneplayer.description = Dient zur Steuerung von unbekannten Sonos Speakern.
+thing-type.sonos.One.label = One
+thing-type.sonos.One.description = Dient zur Steuerung von Sonos One Speakern.
+
+thing-type.config.sonos.zoneplayer.udn.label = Eindeutiger Name (UDN)
+thing-type.config.sonos.zoneplayer.udn.description = Der UDN zur Identifizierung des Sonos Ger�tes.
+thing-type.config.sonos.zoneplayer.notificationTimeout.label = Timeout f�r Benachrichtigungen
+thing-type.config.sonos.zoneplayer.notificationTimeout.description = Timeout f�r Benachrichtigungen (in Sekunden).
+thing-type.config.sonos.zoneplayer.refresh.label = Abfrageintervall
+thing-type.config.sonos.zoneplayer.refresh.description = Intervall zur Abfrage des Sonos Ger�tes (in Sekunden).
+
+# channel types
+channel-type.sonos.add.label = Hinzuf�gen
+channel-type.sonos.add.description = F�gt den aktuellen Sonos Speaker zur aktuellen Gruppe hinzu.
+channel-type.sonos.alarm.label = Alarm aktivieren
+channel-type.sonos.alarm.description = Erm�glicht die Aktivierung des Alarms. Alarm Einstellung m�ssen in der Sonos Controller App erfolgen.
+channel-type.sonos.alarmproperties.label = Alarm Einstellungen
+channel-type.sonos.alarmproperties.description = Zeigt die Einstellungen des aktuellen Alarms an.
+channel-type.sonos.alarmrunning.label = Alarm aktiviert?
+channel-type.sonos.alarmrunning.description = Zeigt an, ob der Alarm aktiviert ist.
+channel-type.sonos.clearqueue.label = Wiedergabeliste leeren
+channel-type.sonos.clearqueue.description = Leert die aktuelle Wiedergabeliste.
+channel-type.sonos.control.label = Fernbedienung
+channel-type.sonos.control.description = Erm�glicht die Steuerung der Wiedergabe von Medien auf dem Sonos Speaker (z.B. Play, Pause).
+channel-type.sonos.coordinator.label = Master
+channel-type.sonos.coordinator.description = Zeigt den UDN des Masters der aktuellen Gruppe an.
+channel-type.sonos.currentalbum.label = Album
+channel-type.sonos.currentalbum.description = Zeigt das Album des aktuellen St�cks an.
+channel-type.sonos.currentalbumart.label = Coverbild
+channel-type.sonos.currentalbumart.description = Zeigt das Coverbild des aktuellen St�cks an.
+channel-type.sonos.currentalbumarturl.label = Coverbild URL
+channel-type.sonos.currentalbumarturl.description = Zeigt die URL zum Coverbild des aktuellen St�cks an.
+channel-type.sonos.currentartist.label = K�nstler
+channel-type.sonos.currentartist.description = Zeigt den K�nstler des aktuellen St�cks an.
+channel-type.sonos.currenttitle.label = Titel
+channel-type.sonos.currenttitle.description = Zeigt den Titel des aktuellen St�cks an.
+channel-type.sonos.currenttrack.label = Track
+channel-type.sonos.currenttrack.description = Zeigt den Track des aktuellen St�cks an.
```

Comment:
Should this be more specific be "Track-Nummer"?

Reply:
I agree. But it is a String item. We should translate it more closely to the english description. It should be `Zeigt den Namen des aktuellen Tracks oder des aktuellen Radiosenders an.`.